src/main/client/Client.ts

```diff
@@ -0,0 +1,6 @@
+import {AxiosResponse} from "axios";
+
+export interface Client {
```

Comment:
Make sure the naming convention in TS is without "I" for interfaces. I believe it needs to be IClient ( that says that it's an interface)